.travis.yml

```diff
@@ -37,7 +37,7 @@ after_script:
 services:
         - redis-server
         - docker
-
+        - riak
```

Comment:
Riak is started in docker now so there is no need to specify it as a service on travis. Moreover some time ago riak was not available on travis as a service on the infrastructure we use now.
In short, I suggest removing this line :)